.github/workflows/build-macos.yml

```diff
@@ -87,21 +87,20 @@ jobs:
 
       # add missing distutils package to python 3.12
       - name: Install distutils
+        run: brew install python-setuptools
```

Comment:
Is this working both for nodejs 18 and 20 as well? I remember that during build it complains about missing distutils

Reply:
I removed it because it didn't work on 18 with the old variant - maybe related to the runner change...